internal/resources/connections/resource_metrics_endpoint_scrape_job.go

```diff
@@ -100,14 +101,14 @@ func (r *resourceMetricsEndpointScrapeJob) Schema(ctx context.Context, req resou
 				},
 			},
 			"name": schema.StringAttribute{
-				Description: "The name of the Metrics Endpoint Scrape Job. Part of the Terraform Resource ID.",
+				Description: "The name of the metrics endpoint scrape job. Part of the Terraform Resource ID.",
 				Required:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"enabled": schema.BoolAttribute{
```

Comment:
I don't see enabled being used elsewhere, such as in the docs. Is it wired up correctly?

Reply:
(I think it was wired up correctly, but we did find an issue in the API. We've decided to remove the "enabled" field for now. To be implemented later: https://github.com/grafana/cloud-onboarding/issues/7842)